reactor-netty-http/src/test/java/reactor/netty/http/client/HttpRedirectTest.java

```diff
@@ -280,22 +247,18 @@ void testIssue278() {
 		final int serverPort2 = SocketUtils.findAvailableTcpPort();
 
 		DisposableServer server1 =
-				HttpServer.create()
+				createServer(serverPort1)
```

Comment:
this test doesn't dispose `server1` and `server2` if assertions fail

Reply:
fixed